src/renderer.ts

```diff
@@ -313,6 +313,23 @@ const richLinkWidth = '8.75rem';
 const richLinkStyles = css`
     background: ${neutral[97]};
     padding: ${basePx(1)};
+    border-top: solid 1px ${neutral[60]};
+    ${textSans.medium()}
+
+    p:last-child {
+        margin-bottom: 0;
+        padding-bottom: 0;
+    }
+
+    p::before {
+        ${icons}
+        content: '\\e005';
+        border-radius: 100%;
+        border: solid 1px ${neutral[7]};
+        padding: 4px 5px 5px 5px;
```

Comment:
The icon font has additional space above the character. Hopefully we can solve this if we move to svgs.